src/autodev.ts

```diff
@@ -75,44 +95,41 @@ const autoDev = async (): Promise<void> => {
   await exec('git fetch')
   await exec(`git config --global user.email "${email}"`)
   await exec(`git config --global user.name "${user}"`)
-  await exec(`git checkout ${branch}`)
-  await exec(`git reset --hard origin/${base}`)
+  const commitDate = await execAndSlurp(
+    `git show -s --format='%ci' origin/${base}`
+  )
+
+  await exec(`git checkout ${base}`)
```

Comment:
Creation of new branch was moved to the end of logic. Its not yet needed, just start on the base branch